packages/nc-gui/components/smartsheet/grid/GroupBy.vue

```diff
@@ -154,18 +153,21 @@ const parseKey = (group: Group) => {
   }
 
   // show the groupBy dateTime field title format as like cell format
-  if (key && group.column?.uidt === UITypes.DateTime && dayjs(key).isValid()) {
-    const dateFormat = parseProp(group.column?.meta)?.date_format ?? dateFormats[0]
-    const timeFormat = parseProp(group.column?.meta)?.time_format ?? timeFormats[0]
-
-    const dateTimeFormat = `${dateFormat} ${timeFormat}`
-
-    return [dayjs(key).utc().local().format(dateTimeFormat)]
+  if (key && group.column?.uidt === UITypes.DateTime) {
+    const dateTime = parseStringDateTime(
+      key,
+      `${parseProp(group.column?.meta)?.date_format ?? dateFormats[0]} ${
+        parseProp(group.column?.meta)?.time_format ?? timeFormats[0]
+      }`,
+    )
+
+    return [dateTime]
```

Comment:
`dateTime` is not used afterwards. you may directly return it.

```ts
return [
  parseStringDateTime(
    key,
    `${parseProp(group.column?.meta)?.date_format ?? dateFormats[0]} ${
      parseProp(group.column?.meta)?.time_format ?? timeFormats[0]
    }`,
  )
]
```

Reply:
done